uint64 timestamp					# time since system start (microseconds)

float32 servo_position					# feedback position of servo. [-1;1]
float32 servo_speed					# feedback speed of servo [deg/s]
float32 servo_acceleration				# feedback of acceleration
uint8 load_pct						# Load percentage
uint8 servo_id						# id of servo
uint8 servo_function					# servo output function (one of Servo1...ServoN)

